### PR TITLE
Add aws installation page

### PR DESCRIPTION
**Pull request by pmech**

### CODE REVIEW

File: content/installation/aws.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Running Humio on AWS
+title: AWS
```
Comment (anagrius) [inline content/installation/aws.md]: The title of the page is used for SEO and should be something that will score high on Google. AWS might is too generic. In any cases having title and menuTitle set to the same value is reduntant, remove menuTitle in that case.

File: content/installation/aws.md [hunk 2]
```diff
@@ -8,18 +8,61 @@ categories_weight: 1
...
+
+The template will create an instance and a data volume and start Humio
+in single-user mode.
+
+When the template in done you can click on the output link to log into
+Humio - give it a few moments to start.
+
+{{< figure src="/pages/installation/aws/template-output.png" >}}
+
+Log-in using user: `developer` and password: "instance ID".
```
Comment (anagrius) [inline content/installation/aws.md]: I would have thought that the password was LITERALLY "instance ID", maybe you can describe that in more detail. E.g. the password is set to the EC2 Instance ID of the node running Humio, you can find it by ...

File: content/installation/aws.md [hunk 3]
```diff
@@ -8,18 +8,61 @@ categories_weight: 1
...
+Humio is available directly from the AWS Marketplace:
 
 https://aws.amazon.com/marketplace/pp/B07DCZKMHQ
 
+## Manual Installation
 
-{{% references %}}
-- [Humio AWS Cloudwatch Logs Integration]({{< ref "aws-cloudwatch.md" >}})
-{{% /references %}}
+You can also choose to manually install Humio on your instances. See
```
Comment (anagrius) [inline content/installation/aws.md]: I would describe this at the top. Something like "While you can install Humio manually by following the "Installing on Bare Metal" guide we suggest you use our Quick Start CloudFormation template to if you are new to Humio.

Comment (mwl) [reply content/installation/aws.md]: I agree with @anagrius.

File: content/installation/aws.md [hunk 4]
```diff
@@ -8,18 +8,61 @@ categories_weight: 1
...
+Humio will listen for HTTP traffic on port 8080, but behind a
+single-user login page. You can restrict access based on IP range if
+wanted. For a production setup we advise you to put a HTTPS proxy in
+front of Humio or place it inside your VPC.
+
+### Sizing
+
+Choosing the right instance size depends on your ingest volume and
+usage patterns. As a general guideline the following table is a
+starting point for sizing your Humio instance.
```
[the remark this answers is not in the document]
Comment (pmech) [reply content/installation/aws.md]: yes. Except that the template do not support all those instance types.

File: content/installation/aws.md (hunk 1 above)
Comment (pmech) [reply content/installation/aws.md]: Done

**Pull request merged**